test/jdk/javax/swing/JPopupMenu/6694823/bug6694823.java

```diff
@@ -60,17 +63,6 @@ public void run() {
             return;
         }
 
-        System.setSecurityManager(new SecurityManager(){
-
-            @Override
-            public void checkPermission(Permission perm) {
-                if (perm.getName().equals("setWindowAlwaysOnTop") ) {
-                    throw new SecurityException();
-                }
-            }
-
-        });
-
         // Show popup as if from an applet
```

Comment:
remove applet

Reply:
Not specific to JEP 486, this should be done as part of a different issue.